### ISSUE TITLE
Insert empty pattern in between patterns

### ISSUE THREAD

**Issue by OnevenHub** [polyend/TrackerIssues#683]
Often when working on a track I find I want to add a new pattern after the previous, but that the next pattern  is already filled up with information. 

Instead of going through each manually and copy/pasting patterns(you now have to manually 'move up' your whole track, it would be great if there was a command that would simply create a new blank pattern in between existing patterns. Would be a great workflow improvement.

**Comment by MetabaronCZ** [comment]
Yes, I would also use that feature. Thank you for posting this request.

**Comment by sunfallz** [comment]
Would be a great feature :)

**Comment by jonwaterschoot** [comment]
Cross posting from the[ lines topic](https://llllllll.co/t/polyend-tracker/30124/703?u=jonwtr ): - adding a link to this request there as well

> _Is there a way to insert a new blank pattern in between already existing patterns?_

> I don’t believe this is possible now.
> A way to reorganize patterns would indeed be very nice.
> My approach is mainly to use the top row for the first ideas, iterations or alternative variations I put on the third and fourth row.
> If it would be possible to select multiple patterns, copy and move them, I agree, this would be cool.
> Of course implementing this would be tricky for the song mode. And who knows what other errors.

**Comment by ryancara** [comment]
I would also love this!
